### ISSUE TITLE
Empty file brings a wrong warning

### ISSUE THREAD

**Issue by sanemat** [jedmao/eclint#28]
Is this expected behavior?
```
# .editorconfig
insert_final_newline = true

$ touch empty.md

## Expected
$ eclint check empty.md
(no warning)
# exit code = 0

## Actual
$ eclint check empty.md
empty.md: expected final newline
# exit code = 1
```

My environments:
```
eclint v0.2.6
node.js v0.12.7
OS X 10.9.5
```

**Comment by jedmao** [comment]
Pretty sure that's expected. I don't think the touch command puts a newline in the file, so it's missing the final newline.

**Comment by sanemat** [comment]
I want to run `$ eclint check "*.md"`. I don't know what to do.

**Comment by sanemat** [comment]
Thanks!
